framework/auth/exceptions.py

```diff
@@ -65,3 +65,7 @@ class MergeConflictError(EmailConfirmTokenError):
     """Raised if a merge is not possible due to a conflict"""
     message_short = language.CANNOT_MERGE_ACCOUNTS_SHORT
     message_long = language.CANNOT_MERGE_ACCOUNTS_LONG
+
+
+class MultipleSSOEmailError(AuthError):
```

Comment:
Minor nit, but would this benefit from having a short comment explaining the error case? The name is pretty informative, in my opinion, but noticed that other error classes in this file have a short comment explanation, so didn't know if this was a convention

Reply:
Yes, will add Docstring for this exception class.